pkgs/applications/radio/uhd/default.nix

```diff
@@ -41,6 +41,8 @@ stdenv.mkDerivation rec {
   # and xxx.yyy.zzz. Hrmpf... style keeps changing
   version = "4.1.0.5";
 
+  outputs = [ "out" "dev" ];
```

Comment:
This is great too, if it doesn't break any dependency.

Reply:
Going to run a nixpkgs-review on this and if that works we are fine.,